array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function showPeopleBelongIn(array, houseName) {
+  const filteredArray = array.filter((person) => person.house === houseName);
+  filteredArray.forEach((person) => {
+    const {firstName, lastName} = person;
+    console.log(`${firstName} ${lastName}`);
+  })
+}
+
+function showTeachersWithPets(array) {
```

Comment:
Same comment here, think a more descriptive name than `array` would be good. Also applies to `filteredArray` below.

Reply:
`array` changed to `people`, `filteredArray` changed to `teachersWithPets`, and `person` changed to `teacherWithPets`.
Thanks.